backend/app/controllers/spree/admin/stock_locations_controller.rb

```diff
@@ -13,6 +13,12 @@ def set_country
         flash[:error] = t('spree.stock_locations_need_a_default_country')
         redirect_to(admin_stock_locations_path) && return
       end
+
+      def collection
+        super.
```

Comment:
Minor style thing: The project seems to use leading periods in multiline method chains over trailing periods. I also think the following two lines are indented too far.

Reply:
I was trying to follow [this one](https://github.com/solidusio/solidus/blob/master/backend/app/controllers/spree/admin/products_controller.rb#L108) that's why I used this style, what do you suggest?

Reply:
Oh weird, I checked a couple of random places and saw the other way. I'm not a big stickler, though I tend to agree with the outcome of [the debate they a had about this in the standardrb issue](https://github.com/testdouble/standard/issues/75). Up to you what you want to do. 

And leading or trailing, I think just indenting it 2 spaces is the way to go.